open_data_schema_map.pages.inc

```diff
@@ -44,8 +42,8 @@ function open_data_schema_map_page_overview() {
   $header[] = t('Endpoint');
   $header[] = t('Output Format');
```

Comment:
This was not introduced by your PR, but the headers for "Endpoint" and "Output Format" are switched. Let's fix that while we're at it.